src/prefect/runner/server.py

```diff
@@ -49,12 +59,60 @@ def _shutdown():
     return _shutdown
 
 
-def start_webserver(
-    runner,
-    log_level: str = None,
-) -> None:
+async def __run_deployment(deployment: "Deployment"):
+    async def _create_flow_run_for_deployment(
+        body: t.Optional[t.Dict[t.Any, t.Any]] = None
+    ) -> JSONResponse:
+        body = body or {}
+        if deployment.enforce_parameter_schema and deployment.parameter_openapi_schema:
+            try:
+                validate_values_conform_to_schema(
+                    body, deployment.parameter_openapi_schema
+                )
+            except ValueError as exc:
+                raise HTTPException(
+                    status.HTTP_400_BAD_REQUEST,
+                    detail=f"Error creating flow run: {exc}",
+                )
+
+        async with get_client() as client:
+            await client.create_flow_run_from_deployment(
+                deployment_id=deployment.id, parameters=body
+            )
+        return JSONResponse(
+            status_code=status.HTTP_201_CREATED, content={"message": "OK"}
```

Comment:
probably useful to return the run ID? not a blocker though

Reply:
added in [ead546e](https://github.com/PrefectHQ/prefect/pull/11333/commits/ead546e6f2f12556b0e72b98c64fb69e8725899f)